### PR TITLE
[NG] Add Nigeria subational data

### PR DESCRIPTION
**Pull request by KhadijaMahanga**
## Description
Adding subnational data
https://docs.google.com/spreadsheets/d/1FDPld2xiG4QJtUqdPL0FT_aue1FPFuFchlB7PznB8xM/edit#gid=647016145

Fixes # (issue)

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Screenshots
Screenshot from 2019-07-15 15-40-18
Screenshot from 2019-07-15 15-53-04
Screenshot from 2019-07-15 17-02-42

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by KhadijaMahanga** [comment]
@kilemensi ready for review

File: takwimu/sql/junior_secondary_school_enrollment.sql [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+level1	NG_1_006	2009	Female	2015	20674
+level1	NG_1_006	2009	Male	2015	21514
+level1	NG_1_006	2009	Female	2016	17472
+level1	NG_1_006	2009	Male	2016	17879
+level1	NG_1_007	2009	Female	2015	183168
+level1	NG_1_007	2009	Male	2015	234728
+level1	NG_1_007	2009	Female	2016	71648
+level1	NG_1_007	2009	Male	2016	83618
+level1	NG_1_008	2009	Female	2015	NA
+level1	NG_1_008	2009	Male	2015	NA
```
Comment (kilemensi) [inline takwimu/sql/junior_secondary_school_enrollment.sql]: These are supposed to be numbers, no?

Comment (KhadijaMahanga) [reply takwimu/sql/junior_secondary_school_enrollment.sql]: @kilemensi we shouldn't have null entry for total, I will remove the fields completely

File: takwimu/profiles/census.py [hunk 2]
```diff
@@ -1791,15 +1985,15 @@ def get_profile(geo, profile_name, request):
                 func = globals()[function_name]
                 data[section] = func(geo, session, country, level)
                 # get profiles for comparative geometries
-                if not data[section]['is_missing']:
+                if not data[section]['is_missing'] and not data[section]['is_missing'] is None:
```
Comment (kilemensi) [inline takwimu/profiles/census.py]: This `None` stuff is a ticking time bomb... Lets think and come up with an approach that will handle all these corner cases.

**Pull request merged**